src/main/java/com/nike/cerberus/store/ConfigStore.java

```diff
@@ -421,6 +429,13 @@ private Properties generateBaseCmsSystemProperties() {
         return Optional.ofNullable(baseParameters.getAccountAdminArn());
     }
 
+    @Deprecated
+    public Optional<String> deprecatedGetAccountAdminArn() {
```

Comment:
I notice the highlander CLI is already not working with legacy environments earlier today.  Maybe we should make a clean break and make it only compatible with new environments and with restoring back-up from old environments.

Reply:
No, im not sure why this is deprecated.

The admin ARN is used everywhere in the environment where KMS keys are created so that the admin user can manage the KMS keys. im 99% sure this information is passed to the cms props

Reply:
@tlisonbee this will have to be a clean break, which will be a good chance to remove those deprecated stack names from that enum.